.github/workflows/release-wheels.yml

```
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 name: "Release: Upload Wheels to PyPi"
-description: "Manually triggered release workflow for wheels build and upload."
+description: "Manually triggered release workflow for one Python distribution."
```

actionlint (1.7.12)

[error] unexpected key "description" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)